.travis.yml

```diff
@@ -33,6 +33,13 @@ before_install:
 
 jobs:
   include:
+    - stage: clang-format
+      if: type = pull_request
+      script:
+        - FORMAT_COMMIT_START=$(echo $TRAVIS_COMMIT_RANGE | awk -F'\\\\.\\\\.' '{print $1}')
```

Comment:
We can make `make format` command in keystone-runtime as well ([SDK now has it](https://github.com/keystone-enclave/keystone-sdk/blob/b84962c3acb6688d1bb66aaf974d08c3751546e9/CMakeLists.txt#L91)), and have it return error code when there are any formatting violations. 
e.g., could you change the Makefile to include these commands?

Reply:
One thing about using `git clang-format` is that we only need to format the hunks that are changed by a certain PR. So we don't need to have a big reformatting commit (that would probably cause all sorts of merge conflicts in pending changes) if we don't want to.